pkg/net/dns.go

```diff
@@ -11,6 +11,8 @@ func NewResolver() *net.Resolver {
 		PreferGo:     true,
 		StrictErrors: true,
 		Dial: func(ctx context.Context, network, address string) (net.Conn, error) {
+			_ = network
+			_ = address
```

Comment:
Wow I hate this 🤣

Reply:
We can also do 

```
func(ctx context.Context, _, _ string) (net.Conn, error) {
}
```

But I thought then nobody knows about the order of network vs address 😓